### ISSUE TITLE
Always rerun particular script

### ISSUE THREAD

**Issue by pun-ky** [valtech/aem-easy-content-upgrade#27]
Is there any way to somehow mark particular script that it will be always running after package being installed? This is sometimes needed in our migration workflow.

**Comment by gruberrolandvaltech** [comment]
Yes, you can name your script myscript.always.groovy. The ".always.groovy" forces run on each deploy. See also example package for this.

TODO: Document fallback and always selectors.

**Comment by pun-ky** [comment]
Cool 😉
